app/editor/src/components/layout/DefaultLayout.tsx

```diff
@@ -25,8 +25,18 @@ export const DefaultLayout: React.FC<ILayoutProps> = ({ name, children, ...rest
   const keycloak = useKeycloakWrapper();
   useToastError();
 
+  const state = React.useContext(SummonContext);
   keycloak.instance.onTokenExpired = () => {
-    keycloak.instance.logout();
+    keycloak.instance
+      .updateToken(86400)
```

Comment:
What's the number represent?

Reply:
The number represents `minValidity` seconds.